src/utils/errors.ts

```diff
@@ -1,22 +1,11 @@
 import { parseJSONRPCError } from './parsing'
 
-export const EVM_ERROR_CODES = [
-  // JSON RPC Standard errors
-  '-32',
-  // Execution reverted error
-  '3',
-]
-
 // EVM clients rely on JsonRpc standard, so we check for those errors.
 export function isEVMError(response: string): boolean {
-  return isError(response, EVM_ERROR_CODES)
-}
-
-function isError(response: string, errorsToCheck: string[]): boolean {
   try {
     const { code } = parseJSONRPCError(response)
 
-    return errorsToCheck.some((error) => String(code).includes(error))
+    return code === 3 || code < -32000
```

Comment:
maybe for clarity in the future since this isn't documented in the JSON RPC standard, something like

```js
if (code === 3) { // execution error
   return true
} elif (code < -32000) { // -32000 itself can be thrown for what are server errors, all other -32000 are user errors 
    return true
} return false
```

Or just some way of making it known why -32000 gets an excpetion